### PR TITLE
Remove ToneJS types repo dep, use library typescript branch instead

### PR DESCRIPTION
**Pull request by panmichal**
The specified ToneJS version points to the typescript branch.  

Should solve https://github.com/MusicAsCode/set-player/issues/3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,7 +27,6 @@
   "devDependencies": {
     "@types/jest": "^23.3.9",
     "@types/node": "^10.12.5",
-    "@types/tone": "https://github.com/Tonejs/TypeScript.git",
```
Comment (styk-tv) [inline package.json]: You mentioned TypeScript definitions come from TypeScript branch. I don't see you select the branch anywhere here? Can you explain?

**Comment by styk-tv** [comment]
Witam, PanMichal.

I need to add TravisCI auto builds so we're looking at the same output when verifying PR. 

I merged both RP's a bit blindfolded and fighting with this now although seems a bit of a red herring https://github.com/fsevents/fsevents/issues/246 Probably completely unrelated when versions match.

File: package.json (hunk 1 above)
Comment (panmichal) [reply package.json]: I changed Tone version to `14.5.23`. 

Here https://www.npmjs.com/package/tone this version is tagged as "typescript" (under "512 versions" tab)

**Comment by panmichal** [comment]
Hi Peter!
When do you get that error? I didn't do anything special: just `npm install`. It did compile `fsevents` without errors.
 Then tested with `npm run build`, `npm run start`.
I'm running Node `10.15.1`.

**Comment by styk-tv** [comment]
I'm running `v13.5.0` need to check what is highest supported by fsevents. Will add tests in travis. finishing up.

**Comment by styk-tv** [comment]
@panmichal all PR's should now build in travis for verification, i triggered first manually, started with 4 node versions 10 11 12 13, we can drop some if we decide so. Check this out.

https://travis-ci.org/MusicAsCode/set-player

File: package.json (hunk 1 above)
Comment (styk-tv) [reply package.json]: Ah, that's how it works. Thank you for showing me this.

**Comment by styk-tv** [comment]
yarn issue with optional dependencies (mac yes, linux no)
yarnpkg/yarn#6834

related to enforced fsevents version
 master  v0.2

switched travis to run build on mac because of above bug

**Comment by panmichal** [comment]
Great with Travis! 
I've updated my Node version to 13. Maybe it's not worth supporting 3 major versions back when 13.* is LTS.

**Pull request merged**